### DISCUSSION TITLE
Side panel instead of popup would be great

### DISCUSSION THREAD

**Discussion by erlendorf** [manugarg/pagenotes-issues#49] [Ideas]
Let the user choose between showing notes in a [side panel ](https://developer.chrome.com/docs/extensions/reference/sidePanel/)and the popup.

**Comment by manugarg** [comment]
Adding it to feature requests. See #74.

**Comment by manugarg** [comment]
I just published a new version (3.5.0) with the sidepanel support. Just right-click on the extension icon and select "Open Side Panel". Please give it a try.

Note - This feature doesn't work on Firefox as Firefox doesn't seem to support the sidepanel API/

In the next release, I'll add an option to open notes either in popup or sidepanel by default.
